File: Python/wej-wyj.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
#
#  szablon.py

def main(args):
    a = str(input('Jak masz na imię?: '))
    b = str(input('Jak masz na nazwisko?: '))
    
    print("Witaj", a,b)
    
    return 0
    
# duck typing

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/wej-wyj.cpp
/*
 * szablon.cpp
 */
using namespace std;

#include <iostream>
#define MAKS 100

int main(int argc, char **argv)
{
    char imie[MAKS];
    char nazwisko[MAKS];
    
    cout << "Podaj imię: " << endl;
    cin.getline(imie, MAKS);
    cout << "Podaj nazwisko: " << endl;
    cin.getline(nazwisko, MAKS);
    cout << "Witaj " << imie <<" "<<nazwisko;
    
	return 0;
}

